antlr-kotlin-runtime/src/commonMain/kotlin/com/strumenta/kotlinmultiplatform/ext/Char.ext.kt

```diff
@@ -35,16 +9,18 @@ public fun Char.Companion.toChars(codePoint: Int, destination: CharArray, offset
   if (isBmpCodePoint(codePoint)) {
```

Comment:
This could become a when. I think it would be clearer and more idiomatic compared to a sequence of ifs